src/miner.cpp

```diff
@@ -71,10 +71,10 @@ bool TxPriorityCompare::operator()(const TxPriority& a, const TxPriority& b)
             const CScCertificate& aCert = dynamic_cast<const CScCertificate&>(*a.get<2>());
```

Comment:
Maybe the comment at lines 64-66 can be updated to reflect new logic; something like:

```
// If we are comparing two certs, we must be sure they are ordered by
// fee rate and, in case of ties, by quality.
// This criterion is a consensus rule and overrides the others two
```